### PR TITLE
Zscaler-FW-Logs

### PR DESCRIPTION
**Pull request by nkanon**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR is created to add modeling rules for Zscaler FW logs.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Zscaler/README.md [hunk 1]
```diff
@@ -3,51 +3,33 @@
...
-
-4. Click **Save.** The NSS server gets added to your ZIA Admin Portal.
-![Add NSS Server](https://raw.githubusercontent.com/demisto/content/9dc21658443d25d7eaf62106b24a2d3c7e9d367d/Packs/Zscaler/doc_files/add_nss_server.png)
+## Collect Events from Vendor
 
-**To download the SSL Certificate:**
-1. Click the **Edit** icon corresponding to the NSS server.
-2. In the **Edit NSS Server** window, under the **SSL Certificate** field, you can either download the SSL Certificate or generate a new certificate for that NSS server. You can upload this SSL certificate to the desired platform.
-![Edit NSS Server](https://raw.githubusercontent.com/demisto/content/9dc21658443d25d7eaf62106b24a2d3c7e9d367d/Packs/Zscaler/doc_files/edit_nss_server.png)
+To configure the Zscaler Internet Access (ZIA) to send logs via the NSS feed output, please refer to **steps 1-3** in the following XDR documentaion which relates to both **Web logs** and **FW logs** [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Logs-from-Zscaler-Internet-Access).
```
Comment (ShirleyDenkberg) [inline Packs/Zscaler/README.md:8]:
```suggestion
To configure the Zscaler Internet Access (ZIA) to send logs via the NSS feed output, refer to steps 1-3 in the following [XDR documentation](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Logs-from-Zscaler-Internet-Access)which relates to both **Web logs** and **FW logs**.
```

File: Packs/Zscaler/README.md [hunk 2]
```diff
@@ -3,51 +3,33 @@
...
+2. [NSS Feed Output Format: Firewall logs](https://help.zscaler.com/zia/nss-feed-output-format-firewall-logs).
+4. [NSS Feed Output Format: Web Logs](https://help.zscaler.com/zia/nss-feed-output-format-web-logs).                                                                                                       
 
-Click and follow the instructions in each of the following links.:
 
-* Web Logs: https://help.zscaler.com/zia/adding-nss-feeds-web-logs
-* SaaS Security Logs: https://help.zscaler.com/zia/adding-nss-feeds-saas-security-logs
+#### Notes:                        
+1. Please make sure to specify the feed escape character as **=**.
+2. As mentioned in the documentation, make sure to add the feed output format for Web logs and/or FW logs.
```
Comment (ShirleyDenkberg) [inline Packs/Zscaler/README.md:19]:
```suggestion
- Make sure to specify the feed escape character as **=**.
- As mentioned in the documentation, make sure to add the feed output format for Web logs and/or FW logs.
```

File: Packs/Zscaler/README.md [hunk 3]
```diff
@@ -3,51 +3,33 @@
...
 You can configure the specific vendor and product for this instance.
 
 1. Navigate to **Settings** > **Configuration** > **Data Broker** > **Broker VMs**. 
 2. Go to the apps tab and add the **Syslog** app. If it already exists, click the **Syslog** app and then click **Configure**.
 3. Click **Add New**.
-4. When configuring the Syslog Collector, set the following values:
-   - vendor as vendor - zscaler
-   - product as product - nssweblog
-
+4. In the **General Settings** section add the following details:
```
Comment (ShirleyDenkberg) [inline Packs/Zscaler/README.md:31]:
```suggestion
4. In the **General Settings** section, add the following details:
```

File: Packs/Zscaler/ReleaseNotes/1_3_13.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Zscaler Internet Access Modeling Rule
+
+- Added modeling rules for FW logs.
```
Comment (ShirleyDenkberg) [inline Packs/Zscaler/ReleaseNotes/1_3_13.md:6]:
```suggestion
Added modeling rules for FW logs.
```

**Comment by ShirleyDenkberg** [comment]
@nkanon Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**